src/parsers/pm_io/psse.jl

```diff
@@ -1443,6 +1449,49 @@ function _psse2pm_multisection_line!(pm_data::Dict, pti_data::Dict, import_all::
     return
 end
 
+function sort_values_by_key_prefix(imp_correction::Dict{String, Any}, prefix::String)
```

Comment:
Don't constrain yourself to a particular `keytype`:
```suggestion
function sort_values_by_key_prefix(imp_correction::Dict{String, <:Any}, prefix::String)
```